js/core/utils/style.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+
+var inflector = require("./inflector");
```

Comment:
Looks like `inflector` is unnecessary variable. 
Can we rewrite it like follows?
```JavaScript
var camelize = require("./inflector").camelize;
```